### ISSUE TITLE
Can't disable PrimaDonnaMethod Smell inline

### ISSUE THREAD

**Issue by adrianbn** [troessner/reek#1216]
There seems to be an issue when trying to disable the PrimaDonnaMethod smell inside a Ruby file as opposed to reek's config file.  I've tried with both reek 4.2.3 and the latest 4.6.2. Take this code example:
```
class Test
  # This method smells of :reek:PrimaDonnaMethod
  def authenticate!
    "stuff"
  end
end
```

Now run reek on it:
```
> reek test-reek.rb
Inspecting 1 file(s):
S

test-reek.rb -- 2 warnings:
  [1]:IrresponsibleModule: Test has no descriptive comment [https://github.com/troessner/reek/blob/master/docs/Irresponsible-Module.md]
  [1]:PrimaDonnaMethod: Test has prima donna method 'authenticate!' [https://github.com/troessner/reek/blob/master/docs/Prima-Donna-Method.md]
```

Am I missing something here? Thanks!

**Comment by mvz** [comment]
You need to place the comment outside the class. This is a little confusing but has to do with how this particular detector is implemented.

**Comment by adrianbn** [comment]
Ah, got it. When I was trying it outside the class I was using the example in https://github.com/troessner/reek/blob/master/docs/Prima-Donna-Method.md and it has a typo, a final semicolon after the smell name. 
It says 
`# :reek:PrimaDonnaMethod: { exclude: [ bravo! ] }` 
where it should say 
`# :reek:PrimaDonnaMethod { exclude: [ bravo! ] }`.

But tested without the extra semicolon and it works with and without an explicit exclusion list, so thank you!

**Comment by troessner** [comment]
>> Ah, got it. When I was trying it outside the class I was using the example in https://github.com/troessner/reek/blob/master/docs/Prima-Donna-Method.md and it has a typo, a final semicolon after the smell name.

Thanks a lot for the pointer, fixed in #1217
